xml/libvirt_overview.xml

```diff
@@ -19,65 +19,393 @@
  </info>
 
  <para>
-  The communication between the virtualization solutions (&kvm;, &xen;)
-  and the libvirt API is managed by the &libvirtd; daemon. It needs to run
-  on the &vmhost;. libvirt client applications such as virt-manager, possibly
-  running on a remote machine, communicate with &libvirtd; running on the
-  &vmhost;, which services the request using native hypervisor APIs. Use the
-  following commands to start and stop &libvirtd; or check its status:
+  A libvirt deployment for accessing KVM or Xen requires one or more daemons
+  to be installed and active on the host. libvirt provides two daemon deployment
+  options: monolithic or modular daemons. By default the monolithic daemon is
```

Comment:
what does monolithic or modular daemons mean?

Reply:
Added more info explaining the two.